cmd/mdatagen/templates/metrics.go.tmpl

```diff
@@ -70,7 +68,6 @@ func (ras *ResourceAttributeSettings) Unmarshal(parser *confmap.Conf) error {
 	if err != nil {
 		return err
 	}
-	ras.enabledProvidedByUser = parser.IsSet("enabled")
```

Comment:
If you remove this line, you don't need custom the unmarshaller anymore. `enabledProvidedByUser` is needed for warnings, but we don't have them in the resource_attriutes yet, so probably can be removed now

Reply:
> we can reuse the metadata.yaml in Config as many receiver implementers have done

I don't understand this part